Tests/Unit/Routing/DynamicRouterTest.php

```diff
@@ -89,6 +91,8 @@ public function testMatchRequest()
      */
     public function testMatchNoRequest()
     {
+        $this->markTestSkipped('Test for deprecated match() method');
```

Comment:
See https://github.com/symfony-cmf/Routing/blob/master/DynamicRouter.php#L218

Should I just remove those tests?